backends/apple/mps/runtime/operations/IndexingOps.mm

```diff
@@ -206,24 +206,27 @@
 
 Error
 MPSGraphBuilder::mpsScatterOp(NodePtr nodePtr) {
-  auto graphNode = nodePtr->mpsnode_union_as_MPSScatter();
-  ET_LOG(
-    Debug, "%s %d: %d",
-    __FUNCTION__, graphNode->input1_id(), graphNode->output_id()
-  );
+  if (@available(iOS 16, macOS 13, *)) {
```

Comment:
Should we return an error for the else statement

Reply:
If the if is not true, then we do not do `_idToMPSGraphTensor[graphNode->output_id()] = ...` which I think would then not set the tensor value and that would fail the prediction, but only if this is used.
Do you think we should return `Error::notSupported` or something like that?

Reply:
Yeah, just we can handle it in a non-fatal way.

Reply:
@cccclai Is the latest version OK?
Thank you for the review :)